ui-common/src/helpers/Utils.ts

```diff
@@ -26,4 +26,20 @@ export class Utils {
 
     return bytes.toFixed(dp) + ' ' + units[u];
   }
+
+static padTo2Digits = (num) => {
```

Comment:
only i see weird indentation?

Reply:
i am not sure that I see it

Reply:
can we fix it?

Reply:
I did format document, I hope it fix what you meant